cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -213,6 +213,7 @@ public static String getZKQuorum(CConfiguration cConf) {
     public static final String REST_PORT = "app.rest.port";
     public static final String PROGRAM_JVM_OPTS = "app.program.jvm.opts";
     public static final String BACKLOG_CONNECTIONS = "app.connection.backlog";
+    public static final String SCAN_APPLICATIONS_BATCHING_CONSUMER_SIZE = "app.scan.application.batching.consumer.size";
```

Comment:
I don't think we need to say "application" here, it's already in "app"

Reply:
This way we can reuse it for other APIs as well

Reply:
Also "consumer" is more of implementation details, skip it. Especially as we use it for transaction size as well